src/tool_parsecfg.c

```diff
@@ -57,25 +62,33 @@ int parseconfig(const char *filename, struct GlobalConfig *global)
 
 #ifndef __AMIGA__
     char *home = homedir();    /* portable homedir finder */
-    filename = CURLRC;   /* sensible default */
+    filename = CURLRC;         /* sensible default */
```

Comment:
I would drop the funny white-space change.

Reply:
No worries - it was to align the two comments.